eslint-plugin-expensify/prefer-at.js

```diff
@@ -0,0 +1,26 @@
+const message = require('./CONST').MESSAGE.PREFER_AT;
+
+module.exports = {
+    meta: {
+        fixable: 'code',
+    },
+
+    create(context) {
+        return {
+            MemberExpression(node) {
+                // Check if we are accessing an array element directly
+                if (node.object.type === 'Identifier' && node.property.type === 'Literal' && typeof node.property.value === 'number') {
```

Comment:
`node.property` is the indexNode right? If so, for expression `arr[arr.length - 1]`, then `node.property.type` shouldn't be `Literal`. Can we detect error for `arr[arr.length - 1]`?